### ISSUE TITLE
setup.py for custom doc_manager

### ISSUE THREAD

**Issue by hannelita** [mongodb-labs/mongo-connector#326]
Hi,
Having a custom doc manager unpacked from Pypi works pretty well if we setup PYTHONPATH. However, would you have any example of s functional setup.py for an external doc manager? I'm having some references issues. 
```
pip install mongo-connector
pip install my-custom-doc-manager
```

When I call mongo-connector with my custom doc manager, it seems that mongo-connector cannot reference my externalised doc manager unless I set a PYTHONPATH on it. ANy ideas?
Thanks in advance!

**Comment by llvtt** [comment]
There shouldn't be a need to set `PYTHONPATH` manually. All that is needed are the two lines in each of the `__init__.py` files from the wiki.

I wrote a quick DocManager implemenation with setup.py file, and this is what the setup.py looks like:

```python
try:
    from setuptools import setup
except ImportError:
    from ez_setup import use_setuptools
    use_setuptools()
    from setuptools import setup

setup(
    install_requires=['mongo-connector >= 2.1'],
    packages=["mongo_connector", "mongo_connector.doc_managers"]
)
```

One thing I ran into when using virtualenv, though, was that when I installed mongo-connector through doing a `python setup.py install` in my new DocManager project directory, that my shell was still using the `mongo-connector` script that I have installed system-wide, not the one in my virtualenv. This doesn't load the new DocManager correctly, so I had to do a `deactivate` then a `source my-doc-manager/bin/activate` again. Then my shell used the correct entry point script.
